src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -342,24 +342,5 @@ private async Task<SyntaxNode> FormatAsync(SyntaxNode newRoot, Document document
             // We do need a setter
             return true;
         }
-
-        private class UseAutoPropertyCodeAction : CustomCodeActions.SolutionChangeAction
```

Comment:
I thought we had more of these sub-types to work around the internal aspect of `CodeActionPriority`? Is this the only one in the code base right now?

Reply:
We got rid of pretty much all the rest, and use the helper methods now.